supervisor/coresys.py

```diff
@@ -121,13 +122,17 @@ async def init_websession(self) -> None:
         if self._websession:
             await self._websession.close()
 
-        resolver = aiohttp.AsyncResolver()
+        try:
+            resolver = aiohttp.AsyncResolver(loop=self.loop)
+            # pylint: disable=protected-access
+            _LOGGER.debug(
+                "Initializing ClientSession with AsyncResolver. Using nameservers %s",
+                resolver._resolver.nameservers,
+            )
+        except AresError as err:
+            _LOGGER.exception("Unable to initialize async DNS resolver: %s", err)
```

Comment:
We don't want to capture that, right?

Reply:
Or, with the aiodns updated to 3.4.0, we could, eventually removing it later? So we'd learn about other ways it can fail.

Reply:
The aiodns 3.4.0 update got already merged, so initializing should no longer fail in that particular case. But just in case there is some other situation where this happens, I'd like to add this fallback. But I'd also like to know about it.

Reply:
But that was my point - would you learn about it with the current code? I thought only level `CRITICAL` gets logged to Sentry if it's handled (because we configure `LoggingIntegration(level=logging.INFO, event_level=logging.CRITICAL)`). `_LOGGER.exception` only creates `ERROR` log entry, which is lower priority.

Reply:
Uh right, I somehow thought that `exception` makes it to Sentry. I'll make a follow up PR!